src/Application.cc

```diff
@@ -728,10 +728,10 @@ std::vector<std::pair<std::string, std::vector<std::string>>>
   for (auto const &path : this->dataPtr->pluginPaths)
     paths.push_back(path);
 
-  // 3. ~/.ignition/gui/plugins
+  // 3. ~/.gz/gui/plugins
   std::string home;
   common::env(GZ_HOMEDIR, home);
-  paths.push_back(home + "/.ignition/gui/plugins");
+  paths.push_back(home + "/.gz/gui/plugins");
```

Comment:
Here we could tick-tock (i.e. support both `ignition` and `gz` for Garden)

Reply:
https://github.com/gazebosim/gz-gui/pull/454/commits/2d40757457b56309f6cab8c2330ace404481e5ba